packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -80,8 +90,28 @@ export async function beaconHandler(args: BeaconArgs & GlobalArgs): Promise<void
       metricsRegistries,
     });
 
-    if (args.attachToGlobalThis) (globalThis as unknown as {bn: BeaconNode}).bn = node;
+    // dev debug option to have access to the BN instance
+    if (args.attachToGlobalThis) {
+      (globalThis as unknown as {bn: BeaconNode}).bn = node;
+    }
 
+    // Prune invalid SSZ objects every interval
+    const {persistInvalidSszObjectsDir} = args;
+    const pruneInvalidSSZObjectsInterval = persistInvalidSszObjectsDir
+      ? setInterval(() => {
```

Comment:
I feel like logic related to pruning ssz objects + constants should not be defined here, extracting to util might be better, similar to `cleanOldLogFiles`.

Reply:
Refactoring is not worth it right now as it would just make implementation less readable, might consider refactoring later if beach handler growths further